lib/features/login/presentation/login_controller.dart

```diff
@@ -208,9 +227,59 @@ class LoginController extends ReloadableController {
 
   @override
   void handleReloaded(Session session) {
+    SmartDialog.dismiss();
+
     popAndPush(
       RouteUtils.generateNavigationRoute(AppRoutes.dashboard),
-      arguments: session
+      arguments: session,
+    );
+  }
+
+  @override
+  void handleGetSessionFailure(GetSessionFailure failure) {
+    SmartDialog.dismiss();
+    super.handleGetSessionFailure(failure);
```

Comment:
can we have a demo video for this case?

Reply:
Please see below demo:


[demo-2.webm](https://github.com/user-attachments/assets/1bfa4713-fed5-4006-a8c0-ac43c64129be)